.github/workflows/generate_api_client.yaml

```diff
@@ -19,14 +19,14 @@ jobs:
       - uses: actions/setup-node@v2
       - uses: actions/checkout@v2
       - run: ./scripts/fetch_api.sh
-      - run: ./scripts/generate_golang.sh
+      - run: ./scripts/generate_go.sh
       - uses: peter-evans/create-pull-request@v3
         with:
-          title: "Update API client"
+          title: "Update ${{ OPENAPI_FILENAME }} API client"
```

Comment:
Will that work? If it does, great but for me it just passed the raw string like this :D

We should have a display name in the client config IMP and use that instead, or the name from the OpenAPI Spec metadata.

Reply:
Agree. Do you want me to revert those changes and put proper issue for adding this. 

I seen that as kinda bummer to not see API name in title (and email notification) but we have no way to verify this changes right now until someone changes API in one of the repos we watch :D

Reply:
Leave it and see if it works :D